lib/trento/settings/analytics.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Trento.Settings.AnalyticsSettings do
+  @moduledoc """
+  Settings related to analytics.
+  """
+
+  use Ecto.Schema
+  use Trento.Support.Ecto.STI, sti_identifier: :installation_settings
+
+  import Ecto.Changeset
+
+  @type t :: %__MODULE__{}
+
+  @primary_key {:id, :binary_id, autogenerate: true}
```

Comment:
Most probably you want to add the jason encoding to avoid sending internals.
You can add `@derive {Jason.Encoder, except: [:__meta__, :__struct__]}` in top of the `primary_key` line